anthos-onprem-terraform/abm_user_cluster_metallb/versions.tf

```
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_providers {
```

```suggestion
  required_version = ">= v0.15.5, < 1.4" # this line should not change during a release
  required_providers {
```
dev-tools 1.10+ requires `required_version`

---

Fixed in https://github.com/GoogleCloudPlatform/anthos-samples/pull/593/commits/992d08b6cccc1eaab22cfd55ac4eb3fc55ffb4de